Common/src/main/java/at/petrak/hexcasting/client/render/PatternTextureManager.java

```diff
@@ -225,20 +225,28 @@ private static void vertex(Matrix4f mat, Matrix3f normal, int light, VertexConsu
     }
 
     public static ResourceLocation getTexture(List<Vec2> points, String pointsKey, int blockSize, boolean showsStrokeOrder, float lineWidth, boolean useFullSize, Color innerColor, Color outerColor) {
-        if (patternTexturesToAdd.containsKey(pointsKey))
-            return patternTexturesToAdd.get(pointsKey);
+        if (patternTexturesToAdd.containsKey(pointsKey)) {
```

Comment:
nice, i think i *tried* to do something like this but i'm bad at java and minecraft so it didn't work